cpp/unreal_projects/SpearSim/Python/spear/log.py

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import inspect
+import os
+
+def log(*args):
+    current_frame = inspect.currentframe()
+    print(_log_get_prefix(current_frame) + "".join([str(arg) for arg in args]))
+
+def _log_get_prefix(current_frame):
+    return "[SPEAR | " + _get_current_file_abbreviated(current_frame) + ":" + _get_current_line(current_frame) + "] "
+
+def _get_current_file_abbreviated(current_frame):
+    outer_frames = inspect.getouterframes(current_frame)
+    return os.path.basename(outer_frames[1].filename)
+
+def _get_current_line(current_frame):
```

Comment:
Let's move `_get_current_line` and `_log_get_prefix` into our existing `python/spear/log.py` file so we get line number output but don't need to have this code checked in redundantly.